redaxo/src/addons/media_manager/pages/effects.php

```diff
@@ -12,7 +12,7 @@
 if (1 != $sql->getRows()) {
     throw new Exception('Invalid type_id "'. $type_id .'"');
 }
-$typeName = $sql->getValue('name');
+$typeName = $sql->getValue('name', 'string');
```

Comment:
Ich glaube, wir müssen hier nicht mal die Min-Rex-Version in den Addons anheben, da ein zusätzlich übergebener Parameter ja nichts ausmachen sollte, oder?